packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -117,6 +117,12 @@
           "when": "view == salesforce-lightning-components && viewItem == external",
           "group": "inline"
         }
+      ],
+      "commandPalette": [
```

Comment:
Updated this since `SFDX: Open Lightning Component` was part of the command palette in scenarios where the extension do not activate and ends up throwing errors.

Reply:
I don't think we want to key off of sfdx:project_opened here. Otherwise it won't show up in scenarios that it should. Lets just add an activation even for "onCommand:salesforce-lightning-quickopen" instead

Reply:
This is causing that command to show in scenarios where nothing related to lightning is present and throws an error if the user runs it.